src/main/java/io/github/queerbric/inspecio/tooltip/SignTooltipComponent.java

```diff
@@ -105,10 +111,30 @@ public int getWidth(TextRenderer textRenderer) {
 	public void drawText(TextRenderer textRenderer, int x, int y, Matrix4f matrix4f, VertexConsumerProvider.Immediate immediate) {
 		if (this.tooltipMode != SignTooltipMode.FAST)
 			return;
-
-		for (var text : this.text) {
-			textRenderer.draw(text, x, y, this.color.getSignColor(), true, matrix4f, immediate, false, 0, 15728880);
-			y += 10;
+		
+		int signColor = this.color.getSignColor();
+
+		if (glowingText) {
+			int outlineColor;
+			if (this.color == DyeColor.BLACK) {
+				outlineColor = -988212;
+			} else {
+				int r = (int) (NativeImage.getRed(signColor) * 0.4);
+				int g = (int) (NativeImage.getGreen(signColor) * 0.4);
+				int b = (int) (NativeImage.getBlue(signColor) * 0.4);
+
+				outlineColor = NativeImage.getAbgrColor(0, b, g, r);
+			}
+
+			for (var text : this.text) {
+				textRenderer.method_37296(text, x, y, signColor, outlineColor, matrix4f, immediate, 15728880);
```

Comment:
For light values there's a constant for it in `LightmapTextureManager`, probably should use that instead.
Will do that myself.